setup/pi/create-backingfiles.sh

```diff
@@ -92,7 +92,7 @@ function add_drive () {
     mkdir "$mountpoint"
   fi
   sed -i "\@^$filename .*@d" /etc/fstab
-  echo "$filename $mountpoint vfat utf8,noauto,users,umask=000,offset=$partition_offset 0 0" >> /etc/fstab
+  echo "$filename $mountpoint vfat utf8,noauto,users,umask=000,discard,offset=$partition_offset 0 0" >> /etc/fstab
```

Comment:
is adding the "discard" option a good idea? `man mount` says it "may have a significant performance impact"
It also seems like it's not necessary because of the manual fstrim above.

Reply:
For some reason I got the impression (as in I thought I saw it not work) that the fstrim did not "push down" into the XFS layer without the discard mount option. I can't see in the vfat code where that would be true. I will test and verify.

The two vfat partitions are only mounted while doing the archive so I don't think the performance impact will be that severe.

Again - Let me test on my setup if the discard is needed and I'll report back.

Reply:
Removed this. The fstrim is pushing down to the xfs layer without it. Not sure what I saw before.